aire/src/card/card.ts

```diff
@@ -24,7 +22,5 @@ export class AireCard {
       //TODO handle uk-card-small and uk-card-large
 
       this.title = this.element.getAttribute("title");
```

Comment:
Isn't this what `@bindable` is for? -- if `@bindable` isn't working create a private field and use accessors.